pathd/path_pcep.c

```diff
@@ -476,14 +521,20 @@ int pcep_cli_pcc_config_write(struct vty *vty)
 		for (int i = 0; i < MAX_PCC; i++) {
 			struct pce_opts *pce_opts = pcep_g->pce_opts[i];
 			if (NULL != pce_opts) {
-				if (PCEP_DEFAULT_PORT != pce_opts->port)
+				if (PCEP_DEFAULT_PORT != pce_opts->port) {
 					csnprintfrr(buff, sizeof(buff),
 						    " port %d", pce_opts->port);
-				if (true == pce_opts->draft07)
-					csnprintfrr(buff, sizeof(buff),
-						    " sr-draft07");
-				vty_out(vty, " pce ip %pI4%s\n",
-					&pce_opts->addr, buff);
+				}
+				if (true == pce_opts->draft07) {
+					csnprintfrr(buff, sizeof(buff), " sr-draft07");
+				}
+				if (IS_IPADDR_V6(&pce_opts->addr)) {
+					vty_out(vty, " pce ip %pI6%s\n",
```

Comment:
This doesn't match the command (`ipv6`), but it should be correct if the command is changed as indicated previously.

Reply:
Ok, I'll fix this.